AdvectionDiffusion.C

```diff
@@ -17,6 +17,10 @@
 #include "Utilities.h"
 #include "Vec3Oper.h"
 #include "AnaSol.h"
+#include "EqualOrderOperators.h"
+
+using WeakOps = EqualOrderOperators::Weak; //!< Convenience renaming
+using ResidualOps = EqualOrderOperators::Residual; //!< Convenience renaming
```

Comment:
these were scoped on purpose. not crucial for this integrand, but i value similar code above the very small namespace pollution reduction this offers. i rely on the scope it in the stokes/nstokes/boussinesq - it is the abstraction layer for the discretization.